website_menu_icon/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,20 @@
+Go to each `Website`, click `Edit` and click on menu item. The confirmation
```

Comment:
This is RST and code is double-backticked:

```
Go to each ``Website``, click ``Edit`` and click on menu item. The confirmation
```



However... This isn't code, right? Then you better use italics:

```suggestion
Go to each *Website*, click *Edit* and click on menu item. The confirmation
```

The same in the rest of the fragments.